pkg/api/v20220904/openshiftcluster_validatestatic_test.go

```diff
@@ -850,7 +850,7 @@ func TestOpenShiftClusterStaticValidateDelta(t *testing.T) {
 		},
 		{
 			name:    "version change",
-			modify:  func(oc *OpenShiftCluster) { oc.Properties.ClusterProfile.Version = "4.3.999" },
+			modify:  func(oc *OpenShiftCluster) { oc.Properties.ClusterProfile.Version = "4.3.99" },
```

Comment:
q: this test was not a signal to support three digit PATCH versions, correct? I see the regex added only allows for 2 digits

Reply:
The test is validating that the version is not changed and has no dependence on the 3 digit z-stream. IMO we would never encounter such a long z-stream in a supported OpenShift version and 2 is more than enough. That said, there is no harm is allowing the extra digit, because the reason for the check is input sanitation. The request would fail the version validation test if wasn't supported anyway, so I'll add it and revert my change to the existing test.

Reply:
Switched to accepting 3 digits from the z-stream.